File: 1._Belajar_JavaScript_Console/1._Latihan_Variasi/Tes_Input_Nested.js
// Tes aja latihan

console.log ("\n Tes aja latihan \n")


var a = Number (prompt ("1 : "))
var b = Number (prompt ("2 : "))
var c = Number (prompt ("3 : "))
var d = Number (prompt ("4 : "))


if ((a > 0) && (b > 0) && (c > 0) && (d > 0)) {
    
    if (a == b == c == d) {
        console.log ("Tes 1")
    }
    
    else if ((a == b) || (b == c) || (c == d) || (a == d)) {
        console.log ("Tes 2")
    }
    
    else {
        console.log ("Bukan Tes 1")
    }
}

else {
    console.log ("Bukan Tes 2")
}

File: 1._Belajar_Python/1._Latihan_Variasi/Tes_Input_Nested.py
# Tes aja latihan

print ("\n Tes aja latihan \n")


a = int (input ("1 :"))
b = int (input ("2 :"))
c = int (input ("3 :"))
d = int (input ("4 :"))


if a > 0 and b > 0 and c > 0 and d > 0:
    
    if a == b == c == d:
        print ("Tes 1")
        
    elif ((a == b) or (b == c) or (c == d) or (a == d)):
        print ("Tes 2")
        
    else:
        print ("Bukan Tes 1")
        
else:
    print ("Bukan Tes 2")